src/python/pants/backend/python/rules/create_requirements_pex.py

```diff
@@ -51,15 +53,20 @@ def create_requirements_pex(request, pex_bin, python_setup, pex_build_environmen
     argv.extend(["--entry-point", request.entry_point])
   argv.extend(interpreter_constraint_args + list(request.requirements))
 
-  execute_process_request = ExecuteProcessRequest(
-    argv=tuple(argv),
-    env=env,
-    input_files=pex_bin.directory_digest,
-    description=f"Create a requirements PEX: {', '.join(request.requirements)}",
-    output_files=(request.output_filename,),
+  execute_process_request = MultiPlatformExecuteProcessRequest(
+    (
+      (Platform.current, Platform.current),
```

Comment:
Kwargs would help here. Also why is Platform.currrent repeated?

Reply:
As described [here](https://docs.google.com/document/d/11dj_OSCooXfIUREMAc-LSZ6MyPPZFEVBYWxncp_xbb8/edit#) the key of the request mapping is a tuple representing that execution and target platform constraints of the requests which it maps to. The way to read this is that the execution platform is constrained to the current (local) platform and the output is meant for the target platform (second element of the tuple). The target platform constraint is not used by speculation but could be used in future changes to declare that some output must be run on a specific platform.

Reply:
Also, I hear you about kwargs, I am going to adopt stu's suggestion about taking a dict instead of a list of tuples. I think that will make the mapping relationship clear.